assets/src/edit-story/app/story/actions/useDeletePage.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
+// @todo this might need more appropriate naming depending on how the deleting Page logic will be.
+// Delete the currently selected Page.
+function useDeletePage( {
```

Comment:
Should this name be changed to `deleteSelectedPage`? Make sure it clear what this function does when you call it.

Reply:
Yep, fare point. There's a Todo note as well in the comment 2 lines above to make the naming more appropriate -- it will be possible to delete a Page selecting it from the thumbnails in the carousel and clicking "Delete" as well so wasn't sure which name would be the best considering the future actions.

Can def make it `useDeleteCurrentPage` or something like that instead for now.